rest-api-spec/src/main/resources/rest-api-spec/api/indices.shrink.json

```diff
@@ -25,6 +25,10 @@
         "master_timeout": {
           "type" : "time",
           "description" : "Specify timeout for connection to master"
+        },
+        "wait_for_active_shards": {
+          "type" : "string",
+          "description" : "Set the number of active shards to wait for before proceeding with the operation."
```

Comment:
the index being shrunk to, I will amend the description here and above for rollover